packages/dashboard/src/components/pages/about-us/about-us.tsx

```diff
@@ -1,32 +1,14 @@
 import * as React from 'react';
 import { Text } from '@deriv/components';
-import { localize } from '@deriv/translations';
-import Frame from 'Components/elements/frame';
-import { useStores } from 'Stores';
+import TempButtons from 'Components/temp-buttons';
 
 const AboutUs: React.FC = () => {
-    const { config_store } = useStores();
-    const total_previews = 27;
-
     return (
         <div className='dw-about-us'>
-            <Text className='dw-about-us__text dw-about-us__title' as='h1' size='l' color='general' align='center'>
-                {localize('HALL OF FAME')}
-            </Text>
-            <Text
-                className='dw-about-us__text dw-about-us__subtitle'
-                as='p'
-                size='m'
-                color='less-prominent'
-                align='center'
-            >
-                {localize('Our front-end evolution has been powered by these amazing people')}
+            <Text as='p' size='l' color='less-prominent' align='center'>
+                About us
```

Comment:
This text doesn't need translation right? Looks `temp` enough.

Reply:
No need translations, it is just there as a reference and soon it will be removed.
You can see this content by redirecting to /dashboard.